habushu-maven-plugin/src/main/java/org/technologybrewery/habushu/InitializeHabushuMojo.java

```diff
@@ -21,27 +21,35 @@ public class InitializeHabushuMojo extends AbstractHabushuMojo {
     @Override
     public void execute() throws MojoExecutionException, MojoFailureException {
 
-	getLog().info("Validating Poetry-based project structure...");
-	PoetryCommandHelper poetryHelper = createPoetryCommandHelper();
-	poetryHelper.execute(Arrays.asList("check"));
-
-	String currentPythonPackageVersion = poetryHelper.execute(Arrays.asList("version", "-s"));
-	String pomVersion = project.getVersion();
-	String expectedPythonPackageVersion = getPythonPackageVersion(pomVersion, false, null);
-
-	if (!StringUtils.equals(currentPythonPackageVersion, expectedPythonPackageVersion)) {
-	    if (overridePackageVersion) {
-		getLog().info(String.format("Setting Poetry package version to %s", expectedPythonPackageVersion));
-		getLog().info(
-			"If you do *not* want the Poetry package version to be automatically synced with the POM version, set <overridePackageVersion>false</overridePackageVersion> in the plugin's <configuration>");
-		poetryHelper.executeAndLogOutput(Arrays.asList("version", expectedPythonPackageVersion));
-	    } else {
-		getLog().debug(String.format(
-			"Poetry package version set to %s in pyproject.toml does not align with expected POM-derived version of %s",
-			currentPythonPackageVersion, expectedPythonPackageVersion));
-	    }
-
-	}
+        getLog().info("Validating Poetry-based project structure...");
+        PoetryCommandHelper poetryHelper = createPoetryCommandHelper();
+        try {
+            poetryHelper.execute(Arrays.asList("check"));
+        } catch (HabushuException e) {
```

Comment:
Seems to occur when you update the minimum python version in your `pyproject.toml`.  It is easy to remiediate, so we catch the error and have poetry resync `pyrpoject.toml` and `poetry.lock`.  If it were to fail during this, the error would be throw to the user.